client/man/epn.conf.5

```diff
@@ -86,6 +86,10 @@ Time to wait, in milliseconds, between each e-mail sent to try to avoid overload
 Specifies the From: e-mail address value in the e-mails sent. The default is noreply@ipadefaultemaildomain. This value can be found by running
 .I ipa config-show
 .TP
+.B mail_from_realname <real name>
+Specifies the From: real name value in the e-mails sent. The default is IPA-EPN. This value can be found by running
+.I ipa config-show
```

Comment:
The value won't show in config-show because it isn't stored in LDAP. It is only in the EPN configuration file (which is fine).

I don't want to bikeshed but I will. Would name be better than real name? It implies there is a fake name.